app/javascript/article-form/components/__tests__/ImageUploader.test.jsx

```diff
@@ -106,9 +106,10 @@ describe('<ImageUploader />', () => {
       },
     });
 
-    expect(await findByText(/uploading.../i)).toBeDefined();
+    expect(await findByText(/uploading.../i)).not.toBeNull();
 
-    waitForElementToBeRemoved(() => queryByText(/uploading.../i));
+    // Upload is finished, so the messsage has disappeared.
+    expect(queryByText(/uploading.../i)).toBeNull();
```

Comment:
At this point the `uploading...` message is gone, so we need to use [queryByText](https://testing-library.com/docs/dom-testing-library/api-queries/#queryby) instead of findByText since it is already removed.